File: emulinker/src/main/java/org/emulinker/util/EmuLinkerXMLConfig.java
package org.emulinker.util;

import org.apache.commons.configuration.*;

public class EmuLinkerXMLConfig extends XMLConfiguration {
  public EmuLinkerXMLConfig() throws ConfigurationException {
    super(EmuLinkerXMLConfig.class.getResource("/emulinker.xml"));
    setThrowExceptionOnMissing(true);
  }
}

File: emulinker/src/main/java/org/emulinker/util/EmuLinkerXMLConfig.kt
package org.emulinker.util

import org.apache.commons.configuration.XMLConfiguration

class EmuLinkerXMLConfig :
    XMLConfiguration(EmuLinkerXMLConfig::class.java.getResource("/emulinker.xml")) {
  init {
    isThrowExceptionOnMissing = true
  }
}
